README.md

```diff
@@ -2,6 +2,9 @@
 
 [![Kotlin Beta](https://kotl.in/badges/beta.svg)](https://kotlinlang.org/docs/components-stability.html)
 [![JetBrains official project](https://jb.gg/badges/official.svg)](https://github.com/JetBrains#jetbrains-on-github)
+[![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.dokka/org.jetbrains.dokka.gradle.plugin?label=MavenCentral&logo=apache-maven)](https://search.maven.org/artifact/org.jetbrains.dokka/org.jetbrains.dokka.gradle.plugin)
+[![Gradle Plugin](https://img.shields.io/gradle-plugin-portal/v/org.jetbrains.dokka?label=Gradle&logo=gradle)](https://plugins.gradle.org/plugin/org.jetbrains.dokka)
+[![License](https://img.shields.io/github/license/Kotlin/dokka.svg)](LICENSE)
```

Comment:
as a side note: the license badge somewhat duplicates the information in the right menu

![2023-03-27_21-01-15](https://user-images.githubusercontent.com/22685910/228040692-a0b76b42-c1b9-40d7-940d-df22edcff124.png)

Reply:
Yeah, I can remove it if you want.

Reply:
I can be nice to include badges and a link to the project in READMEs because there are 3rd party sites that re-host repos, so the GitHub sidebar isn't present e.g. 
https://sourcegraph.com/github.com/Kotlin/dokka

Reply:
Good catch!

Reply:
Oh yeah, true that